src/System.Linq.Expressions/src/System/Linq/Expressions/Interpreter/Utilities.cs

```diff
@@ -15,6 +15,7 @@
 
 namespace System.Linq.Expressions.Interpreter
 {
+#if FEATURE_MAKE_RUN_METHODS
```

Comment:
Ok, thanks.  If these flags already exist and the code being guarded by them here is part of the same feature guarded by the flag elsewhere, I'm fine with this if @VSadov is.